rosa_tutorials/rosa-mobb-cloudwatch-sts.adoc

```diff
@@ -0,0 +1,302 @@
+:_content-type: ASSEMBLY
+[id="rosa-mobb-cloudwatch-sts"]
+= Tutorial: Configuring the Cluster Log Forwarder for Cloudwatch logs and STS
+include::_attributes/attributes-openshift-dedicated.adoc[]
+:context: rosa-mobb-cloudwatch-sts
+
+toc::[]
+
+//Mobb content metadata
+//Brought into ROSA product docs 2023-09-18
+//---
+// date: '2022-08-19'
+// date:
+// title: Configuring the Cluster Log Forwarder for CloudWatch Logs and STS
+// tags: ["AWS", "ROSA"]
+// authors:
+//   - Paul Czarkowski
+//   - Connor Wooley
+// ---
+
+This guide shows how to deploy the Cluster Log Forwarder Operator and configure it to use STS authentication to forward logs to CloudWatch.
+
+[id="rosa-mobb-cloudwatch-sts-prerequisites"]
+== Prerequisites
+
+* A ROSA cluster (configured with STS)
+* The `jq` cli command
+* The `aws` cli command
+
+[id="rosa-mobb-cloudwatch-sts-environmental-setup"]
+== Environment Setup
+
+* Configure the following environment variables:
++
+[NOTE]
+====
+Change the cluster name to match your ROSA cluster and ensure you are logged into the cluster as an Administrator. Ensure all fields are outputted correctly before moving on.
+====
++
+[source,terminal]
+----
+$ export ROSA_CLUSTER_NAME=$(oc get infrastructure cluster -o=jsonpath="{.status.infrastructureName}"  | sed 's/-[a-z0-9]\{5\}$//')
+$ export REGION=$(rosa describe cluster -c ${ROSA_CLUSTER_NAME} --output json | jq -r .region.id)
+$ export OIDC_ENDPOINT=$(oc get authentication.config.openshift.io cluster -o json | jq -r .spec.serviceAccountIssuer | sed  's|^https://||')
+$ export AWS_ACCOUNT_ID=`aws sts get-caller-identity --query Account --output text`
+$ export AWS_PAGER=""
+$ export SCRATCH="/tmp/${ROSA_CLUSTER_NAME}/clf-cloudwatch-sts"
+$ mkdir -p ${SCRATCH}
+$ echo "Cluster: ${ROSA_CLUSTER_NAME}, Region: ${REGION}, OIDC Endpoint: ${OIDC_ENDPOINT}, AWS Account ID: ${AWS_ACCOUNT_ID}"
+----
+
+[id="rosa-mobb-cloudwatch-sts-prep-aws"]
+== Prepare AWS Account
+
+. Create an IAM policy for OpenShift Log Forwarding:
++
+[source,terminal]
+----
+$ POLICY_ARN=$(aws iam list-policies --query "Policies[?PolicyName=='RosaCloudWatch'].{ARN:Arn}" --output text)
+  if [[ -z "${POLICY_ARN}" ]]; then
+  cat << EOF > ${SCRATCH}/policy.json
+  {
+  "Version": "2012-10-17",
+  "Statement": [
+    {
+       "Effect": "Allow",
+       "Action": [
+           "logs:CreateLogGroup",
+           "logs:CreateLogStream",
+           "logs:DescribeLogGroups",
+           "logs:DescribeLogStreams",
+           "logs:PutLogEvents",
+           "logs:PutRetentionPolicy"
+       ],
+       "Resource": "arn:aws:logs:*:*:*"
+    }
+  ]
+  }
+EOF
+$ POLICY_ARN=$(aws iam create-policy --policy-name "RosaCloudWatch" \
+   --policy-document file:///${SCRATCH}/policy.json --query Policy.Arn --output text)
+   fi
+$ echo ${POLICY_ARN}
+----
+
+. Create an IAM role trust policy for the cluster:
++
+[source,terminal]
+----
+$ cat <<EOF > ${SCRATCH}/trust-policy.json
+  {
```

Comment:
same as the above from { to EOF needs to be at the same indent level as the `$`.